src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -1790,5 +1790,301 @@ class B
 
             Test(code, expected, index: 0, compareTokens: false);
         }
+
+        [WorkItem(528, "github.com/dotnet/roslyn/issues/528")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public void TestIntroduceLocalInExpressionBodiedMethod()
+        {
+            var code =
+    @"using System;
+class T
+{
+    int m;
+    int M1() => [|1|] + 2 + 3 + m;
+}";
+
+            var expected =
+    @"using System;
+class T
+{
+    int m;
+    int M1()
+    {
+        const int {|Rename:V|} = 1;
+        return V + 2 + 3 + m;
+    }
+}";
+
+            Test(code, expected, index: 2, compareTokens: false);
```

Comment:
I noticed this the other day... it'd be nice if we had some better way of picking which fix we want than by the numeric index of it within the list -- as is it implicitly kind-of tests part of the ordering of fixes, and it makes the tests harder to read. Not your fault, just wanted to mention it.